packages/salesforcedx-vscode-apex/src/views/testRunner.ts

```diff
@@ -121,6 +125,16 @@ export class ApexTestRunner {
   }
 
   public async runApexTests(tests: string[], testRunType: TestRunType) {
+    const languageClientStatus = languageClientUtils.getStatus() as LanguageClientStatus;
+    if (!languageClientStatus.isReady()) {
+      if (languageClientStatus.failedToInitialize()) {
+        vscode.window.showInformationMessage(
```

Comment:
Not sure if we should change it to an error message. When the extension is activating and the apex language server fails to start we show it as an error message. What do you think ?

Reply:
Error message would be good i think to indicate that the language server failed to start.